README.md

```diff
@@ -28,30 +29,79 @@ ContribHub is a platform that connects developers with open-source projects that
 - TypeScript
 - Tailwind CSS
 - Supabase
-- GitHub API (Octokit)
+- GitHub API (via Octokit)
 
 ## 🚀 Getting Started
 
-A complete guide to setting up and running the project locally can be found in our [Contributing Guidelines](CONTRIBUTING.md).
+### Prerequisites
+
+Make sure you have the following tools installed:
+
+- Node.js
+- npm or yarn
+- Git
+- A Supabase account
+
+### Setting up Locally
+
+1. **Clone the repository**:
+   ```bash
+   git clone https://github.com/graphicmade/contribhub.git
+   cd contribhub
+   ```
+
+2. **Install dependencies**:
+   ```bash
+   npm install
+   ```
+   or
+   ```bash
+   yarn install
+   ```
+
+3. **Create a `.env` file** by copying the `.env.example` file and adding the necessary API keys and environment variables for Supabase and GitHub:
+   ```bash
+   cp .env.example .env
+   ```
+
+4. **Start the development server**:
+   ```bash
+   npm run dev
+   ```
+   The application should now be running on `http://localhost:3000`.
+
```

Comment:
I think it's better to keep this in the [CONTRIBUTING.md](https://github.com/graphicmade/contribhub/blob/main/CONTRIBUTING.md) file